apps/vr-tests/src/utilities/StoryVariantDecorator.tsx

```diff
@@ -14,7 +14,7 @@ const StoryVariantDecorator: Decorator = (storyFn, context) => {
       {storyFn(context)}
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32962&commitId=33498f587f6c66e8afa767567707701c0337e038&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button Command (compat)</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Command (compat).Root RTL.chromium.png|413| Changed|

</details>


  <details><summary><b>Callout</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom Right Edge RTL.chromium.png|1612515| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|1612515| Changed|

</details>


  <details><summary><b>Checkbox</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.End RTL.chromium.png|1614317| Changed|
|Checkbox.Unchecked RTL.chromium.png|1614230| Changed|

</details>


  <details><summary><b>ChoiceGroup</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ChoiceGroup.With Icons RTL.chromium.png|1611892| Changed|

</details>


  <details><summary><b>ColorPicker</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ColorPicker.Root RTL.chromium.png|1577403| Changed|

</details>


  <details><summary><b>HoverCard</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|HoverCard.Root RTL.chromium.png|1614534| Changed|

</details>


  <details><summary><b>Label</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.Root RTL.chromium.png|1613746| Changed|

</details>


  <details><summary><b>Layer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Layer.Root RTL.chromium.png|1613404| Changed|

</details>


  <details><summary><b>Link</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link.Root RTL.chromium.png|1614009| Changed|

</details>


  <details><summary><b>List</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|List.Root RTL.chromium.png|8339| Changed|

</details>


  <details><summary><b>Overlay</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Overlay.Root RTL.chromium.png|1613833| Changed|

</details>


  <details><summary><b>Persona</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona.Initials RTL.chromium.png|1614062| Changed|
|Persona.Size 100 Extra Large RTL.chromium.png|1607934| Changed|
|Persona.Size 120 RTL.chromium.png|1610486| Changed|

</details>


  <details><summary><b>Pivot</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot.Tabs RTL.chromium.png|120| Changed|
|Pivot.Count And Icon RTL.chromium.png|120| Changed|

</details>


  <details><summary><b>ProgressIndicator</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressIndicator.50 RTL.chromium.png|1612320| Changed|

</details>


  <details><summary><b>Separator</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Separator.Horizontal Start Aligned RTL.chromium.png|1614461| Changed|
|Separator.Horizontal End Aligned RTL.chromium.png|1614461| Changed|
|Separator.Vertical Center Aligned RTL.chromium.png|1614460| Changed|
|Separator.Vertical End Aligned RTL.chromium.png|1614460| Changed|
|Separator.Root RTL.chromium.png|1614461| Changed|
|Separator.Vertical Start Aligned RTL.chromium.png|1614460| Changed|

</details>


  <details><summary><b>Shimmer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shimmer.Circle Gap Line RTL.chromium.png|1614978| Changed|

</details>


  <details><summary><b>Slider</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Root RTL.chromium.png|1612947| Changed|

</details>


  <details><summary><b>Stack</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Stack.Horizontal Stack Item Alignments RTL.chromium.png|1395425| Changed|
|Stack.Horizontal Stack Gap RTL.chromium.png|1578930| Changed|
|Stack.Horizontal Stack Reversed RTL.chromium.png|1578699| Changed|
|Stack.Horizontal Stack Right Aligned RTL.chromium.png|1578699| Changed|
|Stack.Vertical Stack Default RTL.chromium.png|1512466| Changed|
|Stack.Horizontal Stack Wrap RTL.chromium.png|1563095| Changed|
|Stack.Horizontal Stack Default RTL.chromium.png|1578925| Changed|
|Stack.Vertical Stack Item Alignments RTL.chromium.png|1435741| Changed|
|Stack.Vertical Stack Right Aligned RTL.chromium.png|1512457| Changed|
|Stack.Vertical Stack Wrap RTL.chromium.png|1405480| Changed|
|Stack.Vertical Stack Reversed RTL.chromium.png|1512466| Changed|

</details>


  <details><summary><b>Text</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Root RTL.chromium.png|1615640| Changed|

</details>


  <details><summary><b>TextField</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextField.Placeholder RTL.chromium.png|1614478| Changed|
|TextField.Multiline RTL.chromium.png|1614520| Changed|
|TextField.Error RTL.chromium.png|1614510| Changed|
|TextField.Icon RTL.chromium.png|1614429| Changed|
|TextField.Suffix RTL.chromium.png|1614543| Changed|
|TextField.Prefix With Value Disabled RTL.chromium.png|1614572| Changed|
|TextField.Prefix With Value Disabled And Prefix Style Override RTL.chromium.png|1614433| Changed|

</details>


  <details><summary><b>TilesList</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TilesList.Basic RTL.chromium.png|1181| Changed|

</details>


  <details><summary><b>Toggle</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toggle.Unchecked RTL.chromium.png|1613654| Changed|
|Toggle.Checked RTL.chromium.png|1613413| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-PieChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-PieChart.Basic RTL.chromium.png|88| Changed|

</details>


  <details><summary><b>react-charting-SparkLineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SparkLineChart.Basic RTL.chromium.png|3938| Changed|

</details>


  <details><summary><b>react-charting-StackedBarChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-StackedBarChart.Benchmark RTL.chromium.png|218| Changed|
|react-charting-StackedBarChart.Basic RTL.chromium.png|1675| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>